### ISSUE TITLE
sh train.sh 报错

### ISSUE THREAD

**Issue by 18811737901** [PaddlePaddle/DeepSpeech#280]
我的参数
-----------  Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 2
dev_manifest: data/aishell/manifest.dev
init_model_path: None
is_local: 1
learning_rate: 0.05
max_duration: 27.0
mean_std_path: data/aishell/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 50
num_proc_data: 3
num_rnn_layers: 3
output_model_dir: ./checkpoints/aishell
rnn_layer_size: 128
share_rnn_weights: 0
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/aishell/manifest.train
trainer_count: 1
use_gpu: 0
use_gru: 1
use_sortagrad: 1
vocab_path: data/aishell/vocab.txt
运行结果：
I0921 16:48:21.802156  1551 Util.cpp:166] commandline:  --use_gpu=0 --rnn_use_batch=True --log_clipping=True --trainer_count=1 
W0921 16:48:21.802270  1551 CpuId.h:112] PaddlePaddle wasn't compiled to use avx instructions, but these are available on your machine and could speed up CPU computations via CMAKE .. -DWITH_AVX=ON
[INFO 2018-09-21 16:48:21,821 layers.py:2716] output for __conv_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-09-21 16:48:21,822 layers.py:3361] output for __batch_norm_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-09-21 16:48:21,823 layers.py:7533] output for __scale_sub_region_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-09-21 16:48:21,825 layers.py:2716] output for __conv_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2018-09-21 16:48:21,827 layers.py:3361] output for __batch_norm_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2018-09-21 16:48:21,829 layers.py:7533] output for __scale_sub_region_1__: c = 32, h = 41, w = 54, size = 70848
I0921 16:48:22.006682  1551 GradientMachine.cpp:94] Initing parameters..
I0921 16:48:22.224419  1551 GradientMachine.cpp:101] Init parameters done.
F0921 16:48:27.090147  1551 hl_cuda_device.cc:565] Check failed: cudaSuccess == cudaStat (0 vs. 35) Cuda Error: CUDA driver version is insufficient for CUDA runtime version
*** Check failure stack trace: ***
    @     0x7f385e536f8d  google::LogMessage::Fail()
    @     0x7f385e53aa3c  google::LogMessage::SendToLog()
    @     0x7f385e536ab3  google::LogMessage::Flush()
    @     0x7f385e53bf4e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f385e4f48a7  hl_stream_synchronize()
    @     0x7f385e268a5e  paddle::SubSequenceLayer::forward()
    @     0x7f385e134d59  paddle::NeuralNetwork::forward()
    @     0x7f385e1359f3  paddle::GradientMachine::forwardBackward()
    @     0x7f385e513449  GradientMachine::forwardBackward()
    @     0x7f385e0e28cc  _wrap_GradientMachine_forwardBackward
    @           0x4c30ce  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c16e7  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c16e7  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c16e7  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4eb30f  (unknown)
    @           0x4e5422  PyRun_FileExFlags
    @           0x4e3cd6  PyRun_SimpleFileExFlags
    @           0x493ae2  Py_Main
    @     0x7f3884d42830  __libc_start_main
    @           0x4933e9  _start
    @              (nil)  (unknown)
Aborted (core dumped)
Failed in training!
调小了参数依然报错，且用的是CPU为什么还会有Cuda Error: CUDA driver version is insufficient for CUDA runtime version，求大神们解答下

**Comment by hsen-dev** [comment]
一样的问题，你怎么解决的
